test/test-trace-client-api.js

```diff
@@ -0,0 +1,124 @@
+var ServiceManager = require('../index').ServiceManager;
+var assert = require('assert');
+var test = require('tap').test;
+var testCmdHelper = require('./meshctl-helper');
+var util = require('util');
+
+test('Test trace client api', function(t) {
+  function TestServiceManager() {
+  }
+  util.inherits(TestServiceManager, ServiceManager);
+
+  function MockMinkelite() {
+  }
+
+  function shutdown() {
+  }
+  MockMinkelite.prototype.shutdown = shutdown;
+
+  function getMetaTransactions(act, host, pid, callback) {
+    assert.equal(act, 'test-app');
```

Comment:
If this mock is used for assertions then these calls should be tracked so that you can assert that they were called.